src/virtual_dom/vnode.rs

```diff
@@ -153,11 +154,35 @@ impl<A: Into<VNode>> FromIterator<A> for VNode {
 impl fmt::Debug for VNode {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match *self {
-            VNode::VTag(ref vtag) => vtag.fmt(f),
-            VNode::VText(ref vtext) => vtext.fmt(f),
-            VNode::VComp(_) => "Component<>".fmt(f),
-            VNode::VList(_) => "List<>".fmt(f),
-            VNode::VRef(_) => "NodeReference<>".fmt(f),
+            VNode::VTag(ref vtag) => f
+                .debug_struct("VTag")
```

Comment:
Cool! I didn't know of this formatter API :)

I am curious why not add `#[derive(Debug)]` to the `VTag` struct?

Reply:
I don't know why I didn't think that before. I did it and with the previous implementation I get the same log. Anyway now we know that we have also that option if we want to custom the output which I find it useful.

Reply:
Yes, I find it useful too!! :)